### DISCUSSION TITLE
Erro - Meu form nada de Submit

### DISCUSSION THREAD

**Discussion by rianck** [balta-io/forum#629] [Tira Dúvidas] [answered]
Pessoal , @andrebaltieri  estou fazendo o form de create no curso Fullstack mas já virei e nao consigo fazer o formulário dar Submit não acontece nada não da mensagem de erro nada ....o que poderia ser - quando eu troco de OnValidSubmit para somente OnSubmit daí funciona porem perda a validação

 <EditForm Model="@InputModel" OnValidSubmit="OnValidSubmitAsync" class="grid grid-cols-2 gap-x-5 gap-y-2.5 text-base">

e  botão esta dentro do Edit

<button type="submit" value="adicionar despesa" class="button-transition w-[100%] cursor-pointer rounded-lg border-none bg-body px-7 py-3.5 text-base font-semibold text-white hover:bg-primary lg:mb-4">
    adicionar empresa
</button>

não esotu conseguindo .... criei projeto no .net 8

desde ja agradeço

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @rianck, normalmente é a falta do `@RenderMode` na página Blazor.
Posta sempre seu código, detalhes do debug, o que já pesquisou e tentou fazer, assim conseguimos te ajudar melhor e mais rápido!
